pkg/cmd/experimental/router/router.go

```diff
@@ -93,22 +93,22 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out io.Writer)
 			case 1:
 				name = args[0]
 			default:
-				glog.Fatalf("You may pass zero or one arguments to provide a name for the router")
+				fmt.Errorf("You may pass zero or one arguments to provide a name for the router")
```

Comment:
@kargakis thanks, i'll fix, trying to make the errors exposed to users less ugly per @deads2k request, thanks for your patience :)